Sourcery/Models/Method.swift

```diff
@@ -100,4 +119,34 @@ final class Method: NSObject, AutoDiffable {
         self.annotations = annotations
     }
 
+    //
+    required init?(coder aDecoder: NSCoder) {
+
+        self.selectorName = aDecoder.decode(forKey: "selectorName")
+        self.parameters = aDecoder.decode(forKey: "parameters")
+        self.returnTypeName = aDecoder.decode(forKey: "returnTypeName")
+        self.returnType = aDecoder.decode(forKey: "returnType")
+        self.accessLevel = aDecoder.decode(forKey: "accessLevel")
+        self.isStatic = aDecoder.decodeBool(forKey: "isStatic")
+        self.isClass = aDecoder.decodeBool(forKey: "isClass")
+        self.isFailableInitializer = aDecoder.decodeBool(forKey: "isFailableInitializer")
+        self.annotations = aDecoder.decode(forKey: "annotations")
+        self.__parserData = aDecoder.decode(forKey: "__parserData")
```

Comment:
you should mark __parserData not codable, no point in serializing that, it's only used in parsing stage